edgelet/contrib/config/windows/config.yaml

```diff
@@ -65,6 +68,10 @@ provisioning:
 #     identity_cert: "<OPTIONAL PATH TO DEVICE IDENTITY CERTIFICATE HERE. IF UNSPECIFIED, DEVICE CA WILL ISSUE ONE>"
 #     identity_pk: "<OPTIONAL PATH TO DEVICE IDENTITY PRIVATE KEY HERE. IF UNSPECIFIED, DEVICE CA WILL ISSUE ONE>"
 
+# provisioning:
```

Comment:
Needs a comment line describing it, like the other examples have. Same for the Linux `config.yaml`s.